ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -487,6 +509,7 @@ JSONObject toJson() throws JSONException {
         metadata.put(Container.JSON_VERSION_KEY, JSON_VERSION_2);
         metadata.put(CONTAINER_ID_KEY, id);
         metadata.put(CONTAINER_NAME_KEY, name);
+        metadata.putOpt(CONTAINER_LAST_UPDATE_TIME_KEY, lastUpdateTime);
```

Comment:
nit: we know the key and value won't be null, so we can just use metadata.put.

Reply:
Updated.